src/test/java/com/google/firebase/auth/multitenancy/FirebaseTenantClientTest.java

```diff
@@ -341,17 +373,16 @@ private static TestResponseInterceptor initializeAppForTenantManagementWithStatu
     return interceptor;
   }
 
-  private static void initializeAppWithResponses(String... responses) {
-    List<MockLowLevelHttpResponse> mocks = new ArrayList<>();
-    for (String response : responses) {
-      mocks.add(new MockLowLevelHttpResponse().setContent(response));
-    }
-    MockHttpTransport transport = new MultiRequestMockHttpTransport(mocks);
-    FirebaseApp.initializeApp(new FirebaseOptions.Builder()
+  private static TenantManager getRetryDisabledTenantManager(MockLowLevelHttpResponse response) {
```

Comment:
Maybe `createRetryDisabledTenantManager` would be a better name since you are creating a new instance?

Reply:
Done